data_summaries/SeaBench.json

```diff
@@ -0,0 +1,58 @@
+{
+    "seabench": {
+        "Unique Dataset Identifier": "seabench",
+        "Dataset Name": "SeaBench",
+        "Paper Title": "SeaLLMs - Large Language Models for Southeast Asia",
+        "Dataset URL": "https://huggingface.co/datasets/SeaLLMs/Sea-bench/raw/main/question.jsonl",
+        "GitHub URL": "https://github.com/DAMO-NLP-SG/SeaLLMs",
+        "Hugging Face URL": "https://huggingface.co/datasets/SeaLLMs/Sea-bench",
+        "ArXiv URL": "https://arxiv.org/abs/2312.00738",
+        "Semantic Scholar Corpus ID": "265551745",
+        "Collection": "SeaBench",
+        "Languages": [
```

Comment:
I noticed that they have a `lang` column. Could we create one entry, e.g. `seabench-en`, `seabench-vi` for each language? They would be basically the same except `Languages`, `Unique Dataset Identifier`, and `Dataset Filter IDs` would be slightly different for each of the 9 entries.

Reply:
Updated! Although I'm not quite sure if this is what you meant: https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/pull/31/commits/8a00d410eacc07f7940317026e3137abbcf43cb9